### DISCUSSION TITLE
[Datasync Server] PUT method returns successfully but data doesn't change in SQL Server database

### DISCUSSION THREAD

**Discussion by pgdiamond123** [Azure/azure-mobile-apps#612] [Q&A]
Currently, I'm checking the PUT method from mobile to Datasync controller. When I call it, it returns status code is 200 and response body is updated, but data isn't changed in SQL Server database.

I'm using DataSync version 5.0.21 and Microsoft.EntityFrameworkCore version 6.0.13

Please help kindly to support my problem. Thank you.

**Comment by adrianhall** [comment]
You've not given us enoguh information to go on here.  Do you have a reproduction in GitHub that I could take a look at?
